packages/block-library/src/legacy-widget/edit/preview.js

```diff
@@ -78,8 +78,7 @@ export default function Preview( { idBase, instance, isVisible } ) {
 						// TODO: This chokes when the query param is too big.
 						// Ideally, we'd render a <ServerSideRender>. Maybe by
 						// rendering one in an iframe via a portal.
-						src={ addQueryArgs( 'themes.php', {
-							page: 'gutenberg-widgets',
+						src={ addQueryArgs( 'widgets.php', {
```

Comment:
Should we also handle different URLs here?

Reply:
I don't think it's necessary. The iframe intercept code will fire regardless of what WP Admin page is loaded.